roles/sync-site/tasks/main.yml

```diff
@@ -101,24 +104,24 @@
     - "sarl -y 'http://{{ vhost }}.stanford.edu/' ''"
     - "sarl -y 'https://{{ vhost }}.stanford.edu/' ''"
   when:
-    vhost != ""
+    vhost is defined
```

Comment:
Hm, I thought I tried this and it didn't work. Although maybe that was when I was setting it as `vhost=""` as a default in the `group_vars`...

Reply:
That may be.  `vhost != ""` failed for me, so I switched to this model.

Reply:
I like this better. Let's keep it this way.